pkg/command/plugin.go

```diff
@@ -150,7 +155,8 @@ func newListPluginCmd() *cobra.Command {
 				// active contexts, but are not installed.
 				installedContextPlugins, missingContextPlugins, err := getInstalledAndMissingContextPlugins()
 				if err != nil {
-					return err
+					errorList = append(errorList, err)
+					log.Warningf("there was an error while getting installed context plugins, error information: '%v'", err.Error())
```

Comment:
Dependent on if we remove the warning printouts or not (see further comment):
Use the constant `errorWhileGettingContextPlugins` for the warning?

Reply:
@chandrareddyp You don't want to use the constant for this warning?